WpfDesign.Designer/Project/Services/CreateComponentTool.cs

```diff
@@ -220,9 +228,9 @@ protected virtual DesignItem[] CreateItems(DesignContext context)
 		/// <summary>
 		/// Is called to create the item used by the CreateComponentTool.
 		/// </summary>
-		public static DesignItem CreateItem(DesignContext context, Type type)
+		public static DesignItem CreateItem(DesignContext context, Type type, object[] arguments)

```

Comment:
same here, add additional static method

Reply:
done ser.changed to  public static DesignItem CreateItem(DesignContext context, Type type, object[] arguments=null)